src/main/java/com/amazon/dlic/util/RolesUtil.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License").
+ *  You may not use this file except in compliance with the License.
+ *  A copy of the License is located at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  or in the "license" file accompanying this file. This file is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *  express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+
+package com.amazon.dlic.util;
+
+import java.util.ArrayList;
+import java.util.stream.Collectors;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class RolesUtil {
+    private static final Logger log = LogManager.getLogger(RolesUtil.class);
+
+    /*
+     * Split the given rolesObject to one or several roles
+     * We accept String or JSONarray only
+     *
+     * rolesObject should be String if given by roles_key
+     * rolesObject should be String or ArrayList if given by roles_path
+     *
+     * Here are some examples:
+     *
+     *   JWT paylaoad contains:
+     *     "access": {
+     *       "roles": [
+     *         "readall, testrole",
```

Comment:
Why some roles are elements of json array and some are comma separated?

Reply:
> Its just a test added by Andy. If roles_path is an array, it should just take strings elements. if roles_path is string, then split by comma